packages/app/components/ManageTabContent.tsx

```diff
@@ -90,10 +92,47 @@ const ManageTabContent = () => {
                 from: address,
             })
             .then((r: string[]) => {setMAddresses(r)})
-            .catch(console.error)
+            .then(() => {
+                // Get details of all private memberships to populate dropdown
+                if(mAddresses.length > 0) {
+                    setShowLoading(true);
```

Comment:
two nits here:
- I would just use the return value `r`, by combining with the `then` above. In React 18, `mAddresses` I think will still be empty
- `showLoading` is still `true`, so this set here is redundant